image-overview-templates/new-names/core-common-config-bootstrapper-arm64.md

```diff
@@ -0,0 +1,20 @@
+$$./common/warning.md
+
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/core-common-config-bootstrapper/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```

Comment:
This image contains the **arm64** version of the service that seeds the Configuration Provider with the configuration common to all services